### PR TITLE
Revolutionizing Node-to-Node Travel: Introducing Time Calculations Based on Euclidean Distances

### PR DESCRIPTION
**Pull request by vrmaverick**
the absolute distance = mod|x2-x1|+mod|y2-y1|+mod[z2-z1|

### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by cclauss** [comment]
Is this algorithm really unique?
* [Edit Distance](dynamic_programming/edit_distance.py)
* [Min Distance Up Bottom](dynamic_programming/min_distance_up_bottom.py)
* [Haversine Distance](geodesy/haversine_distance.py)
* [Lamberts Ellipsoidal Distance](geodesy/lamberts_ellipsoidal_distance.py)
* [Chebyshev Distance](maths/chebyshev_distance.py)
* [Euclidean Distance](maths/euclidean_distance.py)
* [Manhattan Distance](maths/manhattan_distance.py)
* [Minkowski Distance](maths/minkowski_distance.py)
* [Hamming Distance](strings/hamming_distance.py)
* [Levenshtein Distance](strings/levenshtein_distance.py)

File: maths/absolute_distance.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    """
+    Calculate the absolute Euclidean distance between two 3D coordinates.
+
+    Args:
+        coord1 (list): The first 3D coordinate [x, y, z].
+        coord2 (list): The second 3D coordinate [x, y, z].
+
+    Returns:
+        float: The absolute distance between the two coordinates.
+    """
```
Comment (cclauss) [inline maths/absolute_distance.py:14]:
```suggestion
def calculate_absolute_distance(coord1: list[float], coord2: list[float]) -> float:
    """
    Calculate the absolute Euclidean distance between two 3D coordinates.

    Args:
        coord1: The first 3D coordinate [x, y, z].
        coord2: The second 3D coordinate [x, y, z].

    Returns: The absolute distance between the two coordinates.
    """
```

File: maths/absolute_distance.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    abs_diff_x = abs(coord2[0] - coord1[0])
+    abs_diff_y = abs(coord2[1] - coord1[1])
+    abs_diff_z = abs(coord2[2] - coord1[2])
+    total_absolute_distance = abs_diff_x + abs_diff_y + abs_diff_z
+    return total_absolute_distance
```
Comment (cclauss) [inline maths/absolute_distance.py:19]:
```suggestion
    return abs_diff_x + abs_diff_y + abs_diff_z
```

File: maths/absolute_distance.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    # Calculate final linear velocity
+    final_linear_velocity = math.sqrt(
+        initial_velocity**2 + 2 * acceleration * distance
+    )
+
+    # Calculate time using the final velocity
+    time = (final_linear_velocity - initial_velocity) / linear_acceleration
+
+    return time
```
Comment (cclauss) [inline maths/absolute_distance.py:51]:
```suggestion
    return (final_linear_velocity - initial_velocity) / linear_acceleration
```

File: maths/absolute_distance.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    """
+    Calculate the time it takes to travel a given distance with specified speeds.
+
+    Args:
+        distance (float): The distance to travel.
+        linear_speed (float): Linear speed in units per second (default: 3.5).
+        angular_speed (float): Angular speed in degrees per second (default: 120).
+
+    Returns:
+        float: The time in seconds to travel the given distance.
```
Comment (cclauss) [inline maths/absolute_distance.py:32]:
```suggestion
def calculate_travel_time(distance: float, linear_speed: float = 3.5, angular_speed: float = 120) -> float:
    """
    Calculate the time in seconds it takes to travel a given distance with specified speeds.

    Args:
        distance: The distance to travel.
        linear_speed: Linear speed in units per second (default: 3.5).
        angular_speed: Angular speed in degrees per second (default: 120).

    Returns: The time in seconds to travel the given distance.
```

**Comment by tianyizheng02** [comment]
> Is this algorithm really unique?

@cclauss It's not: it's just Manhattan distance.

```py
abs_diff_x = abs(coord2[0] - coord1[0])
abs_diff_y = abs(coord2[1] - coord1[1])
abs_diff_z = abs(coord2[2] - coord1[2])
total_absolute_distance = abs_diff_x + abs_diff_y + abs_diff_z
```

https://github.com/TheAlgorithms/Python/blob/92fbe60082b782d8b85e9667bd6d7832b5383fa3/maths/manhattan_distance.py#L43

I personally don't really see a purpose in adding this when we already implement the distance algorithm.

**Pull request closed without merging**